docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,258 @@
+- Avoid having loading indicators in your App Store screenshots by intelligently waiting for network requests to be finished
+
+After _snapshot_ successfully captures all the screenshots, it will show you a beautiful HTML page to summarize all the screens that were captured:
+
+![HTML summary page](assets/htmlPagePreviewFade.jpg)
+
+## Getting Started Using UI Tests
+
+_snapshot_ uses the capabilities of Apple's UI Tests to drive interactions with your app. To get familiar with writing UI Tests, check out the following introductions:
+
+- [WWDC 2015 Introduction to UI Tests](https://developer.apple.com/videos/play/wwdc2015-406/)
+- [A first look into UI Tests](http://www.mokacoding.com/blog/xcode-7-ui-testing/)
+- [UI Testing in Xcode 7](http://masilotti.com/ui-testing-xcode-7/)
+- [HSTestingBackchannel : ‘Cheat’ by communicating directly with your app](https://github.com/ConfusedVorlon/HSTestingBackchannel)
+- [Automating App Store screenshots using fastlane snapshot and frameit](https://tisunov.github.io/2015/11/06/automating-app-store-screenshots-generation-with-fastlane-snapshot-and-sketch.html)
+
+To jump-start your UI tests, you can use the UI Test recorder. By interacting with your app through the simulator while using it, Xcode will generate UI Test code for you. You can find the red record button on the bottom of the screen, and there's more information in [this blog post](https://krausefx.com/blog/run-xcode-7-ui-tests-from-the-command-line)
+
+## Setting Up _snapshot_
+
+1. Create a new UI Test target in your Xcode project (See the top part of [this article](https://krausefx.com/blog/run-xcode-7-ui-tests-from-the-command-line))
+1. Run `snapshot init` in your project folder
+1. Add the ./SnapshotHelper.swift file to your UI Test target (You can move the file anywhere you want)
+1. (Objective C only) Add the bridging header to your test class.
+    - `#import "MYUITests-Swift.h"`
+    - The bridging header is named after your test target with -Swift.h appended.
+1. In your UI Test class, click the `Record` button on the bottom left and record your interaction
+1. To take a screenshot, call the following between interactions
+    - Swift: `snapshot("01LoginScreen")`
+    - Objective C: `[Snapshot snapshot:@"01LoginScreen" waitForLoadingIndicator:YES];`
+1. Add the following code to your `setUp()` method:
+
+**Swift**
+
+```swift
+let app = XCUIApplication()
+setupSnapshot(app)
+app.launch()
+```
+
+**Objective C**
+
+```objective-c
+XCUIApplication *app = [[XCUIApplication alloc] init];
+[Snapshot setupSnapshot:app];
+[app launch];
+```
+
+If you have _fastlane_ installed, it's easy to give _snapshot_ a try. First clone the _fastlane_ repo, head over to the _snapshot_ example project, and then run `fastlane snapshot`
+
+```
+$ git clone https://github.com/fastlane/fastlane
+
+$ cd fastlane/snapshot/example
+
+$ fastlane snapshot
+```
+
+![assets/snapshot.gif](assets/snapshot.gif)
+
+# Decorate Screenshots with Device Frames
```

Comment:
I'm not a big fan of decorate, maybe we can use `Put your screenshots into device frames`, or similar? It also has to look nice in the sidebar 👍